content/components/icon-button.mdx

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Hey! This looks great, but I'd add Icon button from Rails too (https://primer.style/view-components/components/beta/iconbutton)

Reply:
Do you know how I find the PVC component ID?

Reply:
PVC components aren't in the new IA yet, but I'm working on it!

Reply:
just need to hardwire that link in 🔗 for now